vanilla/vanilla
# Allow Rich content to be converted if Garden.ForceInputFormatter exists

Editing rich content when `Garden.ForceInputFormatter` is `true` causes unformatted json to be rendered in a discussion or comment if the InputFormatter is set to something other than `Rich`.  This detects if `Garden.ForceInputFormatter` and if the Forced InputFormat is different then the original contents allows for it to be modified.

To Test

1) Set Garden.ForceInputFormatter = true` in config.php
2) post rich discussion and comment
3) Set posting format to TextEx
4) Try edit the discussion and comment
5) Content should save and display properly 

closes https://github.com/vanilla/support/issues/1210
